worker/crates/action-processor/src/plateau/attribute_flattener.rs

```diff
@@ -0,0 +1,721 @@
+                            AttributeValue::String(s) => s,
+                            _ => panic!("Expected AttributeValue::String"),
+                        },
+                        match v {
+                            Some(AttributeValue::String(ref s)) => s,
+                            _ => panic!("Expected Some(AttributeValue::String)"),
+                        }
+                    ))
+                };
+                flattened.insert(name.clone(), value.clone());
+            }
+
+            if ftype == &AttributeValue::String("bldg::BuildingPart".to_string()) {
+                self.flattener
+                    .extract_fld_risk_attribute(feature, root_all_attrib.clone());
+                self.flattener
+                    .extract_tnm_htd_ifld_risk_attribute(feature, root_all_attrib.clone());
+                self.flattener
+                    .extract_lsld_risk_attribute(feature, root_all_attrib.clone());
+
+                // BuildingPart に記述されている災害リスク属性も抽出する。
+                // 規格外ではあるが BuildingPart 下位に災害リスク属性が記述されているデータもあるため。
+                self.flattener
+                    .extract_fld_risk_attribute(feature, all_attrib.clone());
+                self.flattener
+                    .extract_tnm_htd_ifld_risk_attribute(feature, all_attrib.clone());
+                self.flattener
+                    .extract_lsld_risk_attribute(feature, all_attrib.clone());
+            }
+
+            // フラットにする属性の設定
+            for (name, value) in &flattened {
+                feature
+                    .attributes
+                    .insert(Attribute::new(name.clone()), value.clone());
+            }
+
+            self.common_processor.update_max_lod(feature);
+            self.features.push(feature.clone());
+        }
+        Ok(())
+    }
+
+    fn finish(
+        &self,
+        ctx: NodeContext,
+        fw: &mut dyn ProcessorChannelForwarder,
+    ) -> Result<(), BoxedError> {
+        if self.features.is_empty() {
+            return Ok(());
+        }
+
+        // スキーマ定義名
+        let first_feature = &self.features[0];
+        let city_code = match first_feature.attributes.get(&Attribute::new("cityCode")) {
+            Some(AttributeValue::String(s)) => s.clone(),
+            _ => {
+                return Err(PlateauProcessorError::AttributeFlattener(
+                    "No city code found".to_string(),
+                )
+                .into())
+            }
+        };
+        let schema_definition = format!("building_{}", city_code);
+
+        // スキーマ属性を作成してデータフィーチャーに設定、出力
+        let mut attrib = HashMap::new();
+        attrib.insert(
+            "schemaDefinition".to_string(),
+            AttributeValue::String(schema_definition),
+        );
+        attrib.insert(
+            "maxLod".to_string(),
+            AttributeValue::Number(self.common_processor.max_lod.into()),
+        );
+
+        attrib.insert("isFilepathFeature".to_string(), AttributeValue::Bool(false));
```

Comment:
Isn't isFilepathFeature true?

Reply:
https://github.com/reearth/reearth-flow/blob/python-caller-attribute-flattener/worker/crates/action-processor/src/plateau/attribute_flattener.rs#L702
It is set to true here.

Setting it to false here meant that the column was not output to the CSV unless it already existed.

Reply:
My apologies!
Thank you so much!